TrackletAlgorithm/Constants.h

```diff
@@ -3,8 +3,10 @@
 
 #include "ap_int.h"
 
-//#include <cmath>
-//#include "hls_math.h"
+// Macro to convert floating point values to integers, given a digitization
+// constant. The 1.0e-1 is a fudge factor needed to get the floating point
+// truncation to agree exactly with the emulation.
+#define INT(x,k) (static_cast<int>(x / k + 1.0e-1))
```

Comment:
this should be ((x)/((k)+1...) due to the usual issue with preprocessor macros.  or better yet since we are in C++ it should be an inline function.

Reply:
Switched to an inline function.